lgsvl_pkgs/lgsvl_msgs/proto/BUILD

```diff
@@ -0,0 +1,14 @@
+proto_library(
+    name = "lgsvl_lib",
+    srcs = [
+        "detection2d.proto",
+        "detection2darray.proto",
+        "detection3d.proto",
+        "detection3darray.proto",
```

Comment:
As mentioned in https://github.com/ApolloAuto/apollo/pull/11187, prefer slim targets, with 1 proto in `srcs` of a library.

Reply:
Changed.